applications/GeoMechanicsApplication/tests/test_set_multiple_moving_load_process.py

```diff
@@ -1,12 +1,54 @@
 import KratosMultiphysics
-
 import KratosMultiphysics.GeoMechanicsApplication as GMA
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 
-class TestSetMultipleMovingLoadsProcess(KratosUnittest.TestCase):
+
+class KratosGeoMechanicsSetMultipleMovingLoadProcessTests(KratosUnittest.TestCase):
     time_dependent_load = '["0.0", "-2.0*t", "0.0"]'
     time_dependent_velocity = ' "2.0*t" '
 
+    def set_nodal_displacement_dof(self):
+        for node in self.mp.Nodes:
+            node.AddDof(KratosMultiphysics.DISPLACEMENT_X)
+            node.AddDof(KratosMultiphysics.DISPLACEMENT_Y)
+            node.AddDof(KratosMultiphysics.DISPLACEMENT_Z)
+
+    @staticmethod
+    def setup_strategy(mp):
+        """
+        Setup default strategy for solving the problem
+
+        Parameters
+        ----------
+        mp: model part
+
+        Returns default strategy
+        -------
+
+        """
+
+        linear_solver = KratosMultiphysics.SkylineLUFactorizationSolver()
+        builder_and_solver = KratosMultiphysics.ResidualBasedBlockBuilderAndSolver(linear_solver)
+        scheme = KratosMultiphysics.ResidualBasedIncrementalUpdateStaticScheme()
+        convergence_criterion = KratosMultiphysics.ResidualCriteria(1e-1, 1e-1)
+        convergence_criterion.SetEchoLevel(0)
+
+        max_iters = 10
+        compute_reactions = False
+        reform_step_dofs = False
+        move_mesh_flag = False
+        strategy = KratosMultiphysics.ResidualBasedNewtonRaphsonStrategy(mp,
```

Comment:
Do you think we really need a 'real' solving strategy here, or could we get away with creating a 'dummy' one, so we do not depend on all these objects in the test? You probably added the strategy for a reason, so let's discuss if this is a possibility here.

Reply:
Unfortunately we need it to be able to initialize the movingloadcondition at each solution step as it appears the InitializeSolutionStep method although public in c++ is not exposed in python

Reply:
Although we could write some standard dummys in c++ to instantiate in python ...... but is it worth it ?

Reply:
Indeed, creating a dummy in C++, which only has this one override which initializes the movingloadcondition at each step would be the way to get there. If this was a C++ test, it would be a no-brainer, because you can create this dummy locally in your test file, and just add the functionality you need. 

In this case, it is a bit more tricky, because it could happen that people start adding to this separate C++ 'dummy' more and more, until it becomes almost as complex as the real thing. 

My proposal would be to think a bit more about these kinds of re-usable test-utilities, because if it's done right, it will help a lot in making our tests more independent and test only the thing they need to test, meaning they also have a single reason for failure.

For now, this broader discussion should not block this any further, so I approved the PR.

FYI @avdg81